pycbc/tmpltbank/sky_grid.py

```diff
@@ -0,0 +1,118 @@
+"""Functionality for handling grids of points in the sky for coherent SNR
+calculation via `pycbc_multi_inspiral`. The main operation to be performed on
+these points is calculating the antenna pattern functions and time delays from
+the Earth center for a network of detectors.
+"""
+
+import numpy as np
+import h5py
+
+from pycbc.detector import Detector
+
+
+class SkyGrid:
+    def __init__(self, ra, dec):
+        """Initialize a sky grid from a list of RA/dec coordinates.
+
+        Parameters
+        ----------
+        ra: iterable of floats
+            Right ascensions for each point UNITS AND ANGULAR CONVENTION.
+        dec: iterable of floats
+            Declination for each point UNITS AND ANGULAR CONVENTION.
+        """
+        # Question: should the list of detectors and reference times be part of
+        # the SkyGrid object? Or should they be given each time they are needed,
```

Comment:
I think I would store these in the SkyGrid object. I guess the issue if there are other, not PyGRB, use cases for this, but the class could always be refactored to accommodate such needs if this becomes relevant.

Reply:
Sounds good.